src/nni_manager/training_service/pai/paiK8S/paiK8STrainingService.ts

```diff
@@ -185,7 +185,16 @@ class PAIK8STrainingService extends PAITrainingService {
             }
         }
 
-        return yaml.safeDump(paiJobConfig);
+        if (this.paiTrialConfig.paiConfigPath) {
+            try {
+                const additionalPAIConfig = yaml.safeLoad(fs.readFileSync(this.paiTrialConfig.paiConfigPath, 'utf8'));
+                return yaml.safeDump({...paiJobConfig, ...additionalPAIConfig});
+            } catch (error) {
+                this.log.error(`Loading ${this.paiTrialConfig.paiConfigPath} error: ${error}`);
```

Comment:
this error message is not accurate, could be:
error occurs during loading and merge .... : ${error}

Reply:
fixed.